packages/webpack/src/presets/base.ts

```diff
@@ -23,7 +23,7 @@ function baseConfig (ctx: WebpackConfigContext) {
 
   ctx.config = {
     name: ctx.name,
-    entry: { app: [resolve(options.appDir, 'entry')] },
+    entry: { app: [options.experimental.asyncEntry ? resolve(options.appDir, 'entry.async') : resolve(options.appDir, 'entry')] },
```

Comment:
Refactor: we could shorten logic by adding asyncEntry chrck inside template literal:

```suggestion
    entry: { app: [resolve(options.appDir,  options.experimental.asyncEntry? 'entry.async' : 'entry') ] },
```